src/Components/DnDLayout/GridTile.tsx

```diff
@@ -62,6 +64,14 @@ const GridTile = ({ widgetType, isDragging, setIsDragging, setWidgetAttribute, w
     return null;
   }
 
+  const widgetLink = (href: string) => {
+    if (href.includes('https://')) {
```

Comment:
is there a better way to check if the href is an external link?

Reply:
This is not reliable. We use a `isExternal` prop in our links to determine if a link is external. Configuration is responsible for marking external links.

Reply:
I'm not seeing an `isExternal` prop being passed down

Reply:
I don't think exists yet in the widgets. But its a pattern all across the platform.